package.json

```diff
@@ -11,22 +11,23 @@
     "scaffold": "node scripts/create-component.mjs"
   },
   "dependencies": {
-    "next": "12.0.7",
+    "classnames": "^2.3.1",
```

Comment:
"^2.3.1" or "2.3.1"?

Reply:
`npm` did that. I've removed it for consistency.

Reply:
Pro tip: `npm i -E classname`
_-- Claus Wahlers, CLI Wizard Extraordinaire_